src/net/sourceforge/kolmafia/textui/command/GuzzlrCommand.java

```diff
@@ -0,0 +1,90 @@
+package net.sourceforge.kolmafia.textui.command;
+
+import net.sourceforge.kolmafia.KoLCharacter;
+import net.sourceforge.kolmafia.KoLConstants.MafiaState;
+import net.sourceforge.kolmafia.KoLmafia;
+import net.sourceforge.kolmafia.RequestThread;
+import net.sourceforge.kolmafia.objectpool.ItemPool;
+import net.sourceforge.kolmafia.preferences.Preferences;
+import net.sourceforge.kolmafia.request.GenericRequest;
+import net.sourceforge.kolmafia.session.InventoryManager;
+
+public class GuzzlrCommand extends AbstractCommand {
+  public GuzzlrCommand() {
+    this.usage = " [abandon | accept <bronze|gold|platinum>] - Use the Guzzlr tablet";
+  }
+
+  @Override
+  public void run(final String cmd, String parameters) {
+    if (!InventoryManager.hasItem(ItemPool.GUZZLR_TABLET)
+        && !KoLCharacter.hasEquipped(ItemPool.GUZZLR_TABLET)) {
+      KoLmafia.updateDisplay(MafiaState.ERROR, "You don't have a Guzzlr tablet.");
```

Comment:
"You don't have a Guzzlr tablet." or "You don't have a Guzzlr tablet equipped"

It is a nit but a message that tells the user exactly what to fix is more helpful than a suggestion.

Reply:
It can be in your inventory or equipped: either works.